internal/app/connectconformance/server_runner.go

```diff
@@ -251,5 +253,17 @@ func runTestCasesForServer(
 	}
 
 	// If there are any tests without outcomes, mark them now.
-	results.failRemaining(testCases, errors.New("no outcome received from the client"))
+	results.failRemaining(testCases, &errFailedToGetResult{errNoOutcome})
+}
+
+type errCouldNotRun struct {
```

Comment:
This marker type is used to account for tests that were never sent to the client when reporting that in the results. That figure includes all test cases whose outcome is this kind of error as well as all test cases that have no result at all.